app/src/main/java/ca/fuwafuwa/kaku/Windows/CaptureWindow.java

```diff
@@ -56,7 +57,13 @@ public boolean onTouch(MotionEvent e) {
     @Override
     public boolean onDoubleTap(MotionEvent e) {
         tessStarted = true;
-        mTessRunnable.runTess(new BoxParams(params.x, params.y + getStatusBarHeight(), params.width, params.height));
+
+        int[] viewPos = new int[2];
+        mWindowBox.getLocationOnScreen(viewPos);
+
+        // TODO: Replace the first 1 with R.drawable.border_translucent.StrokeWidth when I figure out how
+        int offset = KakuTools.dpToPx(this.context, 1)+1;
+        mTessRunnable.runTess(new BoxParams(viewPos[0]+offset, viewPos[1]+offset, params.width-(2*offset), params.height-(2*offset)));
```

Comment:
Try changing the OCR to happen on these events:

```Java
    @Override
    public boolean onTouch(MotionEvent e) {

        setOpacity(e);

        boolean handled = super.onTouch(e);

        if (e.getAction() == MotionEvent.ACTION_UP){

            int[] viewPos = new int[2];
            mWindowBox.getLocationOnScreen(viewPos);

            tessStarted = true;
            mTessRunnable.runTess(new BoxParams(viewPos[0], viewPos[1], params.width, params.height));
            return true;
        }

        return handled;
    }

    @Override
    public boolean onResize(MotionEvent e) {

        setOpacity(e);

        boolean handled = super.onResize(e);

        if (e.getAction() == MotionEvent.ACTION_UP){

            int[] viewPos = new int[2];
            mWindowBox.getLocationOnScreen(viewPos);

            tessStarted = true;
            mTessRunnable.runTess(new BoxParams(viewPos[0], viewPos[1], params.width, params.height));
            return true;
        }

        return handled;
    }
```

If you do, you should start seeing this bug:

![bug](https://cloud.githubusercontent.com/assets/878158/22049688/e3fe6144-dce9-11e6-947a-0237acd45fa3.png)

The reason why we might want to OCR immediately after the user lets go of the box is in the future, instead of double tapping, we simply have the user move around the box and in whatever free space the screen still has, we will display the results of the OCR (instead of popping up InformationWindow and taking over the ENTIRE screen, which especially isn't a pleasant experience for tablets).